modules/about-oadp-update-channels.adoc

```diff
@@ -11,15 +11,17 @@ When you install an OADP Operator, you choose an _update channel_. This channel
 
 There are three update channels:
```

Comment:
If it is not common practice, consider rewording this to remove 'three'.

```suggestion
The available update channels:
```

If it is...

```suggestion
There are four update channels:
```